services/QuillLMS/client/app/assets/styles/evidence-tool.scss

```diff
@@ -348,6 +348,20 @@
   }
 }
 
+.tutorial-video-container {
+  height: 750px;
+  background-color: #F8F9F9;
+  .video-section video {
+    padding-top: 50px;
+    max-width: 950px;
+  }
+  .video-caption {
+    margin-top: 25px;
+    font-size: 16px;
+    font-weight: 500;
```

Comment:
I don't think `font-weight: 500` actually displays differently from `400` in our typeface.